### PR TITLE
WIP: add label color directly in the core

### LINKED ISSUE #59: feat: Assign labels a color
It would be nice to be able to assign each label a color, and use that color every time the label is printed.

### PR DESCRIPTION
**Pull request by ludovicm67**
Fixes #59.

Add label color directly in the core so that it is consistent within all UI.

### CODE REVIEW

File: bug/label.go [hunk 1]
```diff
@@ -1,19 +1,62 @@
...
 type Label string
 
 func (l Label) String() string {
 	return string(l)
 }
 
+// RGBColor from a Label computed in a deterministic way
+func (l Label) RGBColor() RGBColor {
+	id := 0
+	hash := sha1.Sum([]byte(l))
```
Comment (sladyn98) [inline bug/label.go]: Can we assign the colors for the labels on some fixed domain like a specific color for a particular level ?

**Comment by MichaelMure** [comment]
Blocker gqlgen issue: https://github.com/99designs/gqlgen/issues/670

**Comment by MichaelMure** [comment]
Issue is fixed in gqlgen now, we need to upgrade and merge this.

**Comment by A-Hilaly** [comment]
@ludovicm67 i rebased and added some tests to verify that the new vendors works as expected !

**Comment by sandhose** [comment]
I rebased the branch and added @A-Hilaly's work on the webui

**Pull request merged**